CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ Changes to Calva.
 - [Support for custom project/workflow commands](https://github.com/BetterThanTomorrow/calva/issues/281)
 
 ## [Unreleased]
+- [REPL ns prompt does not change with (ns ..) form](https://github.com/BetterThanTomorrow/calva/issues/280)
```

Comment:
This should reflect what has happened with the change. So either something like what I renamed the PR to, or simply add the word ”Fix” in the beginning.